src/storeinbox.cpp

```diff
@@ -27,7 +28,7 @@ ReturnValue StoreInbox::queryAdd(int32_t, const Thing& thing, uint32_t, uint32_t
 			return RETURNVALUE_CANNOTMOVEITEMISNOTSTOREITEM;
 		}
 
-		const Container* container = item->getContainer();
+		auto container = item->getContainer();
```

Comment:
Could this be `const auto`
```suggestion
		const auto container = item->getContainer();
```